src/animator.js

```diff
@@ -250,8 +258,14 @@ export class VelocityAnimator {
    * @returns Resolved when the animation is done
    */
   addClass(element: HTMLElement, className: string): Promise<boolean> {
```

Comment:
I see that enter/leave do accept an options attribute, is that not needed for remove/add?